Jeonguk_REST_API_DOCS.md

```
@@ -0,0 +1,1039 @@
+
+**GET** `/users/me/posts` 
+
+로그인한 사용자가 쓴 게시글을 조회하여 목록을 보여주는 리소스이다.(로그인 인증)
+
+**Request Headers**
+
+| 헤더 | 타입 | 필수 | 설명 |
+| --- | --- | --- | --- |
+| **Authorization** | string | O | 로그인 한 사용자가 쓴 게시글을 조회하는 리소스이므로 Bearer 토큰 인증이 필요합니다. |
+
+**Path Parameters**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)